packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPReceiptManager.m

```diff
@@ -14,13 +14,13 @@ @interface FIAPReceiptManager ()
 // Gets the receipt file data from the location of the url. Can be nil if
 // there is an error. This interface is defined so it can be stubbed for testing.
 - (NSData *)getReceiptData:(NSURL *)url error:(NSError **)error;
-
+@property(strong, nonatomic) NSURL *recieptURL;
```

Comment:
This should be readonly.

Reply:
Also should add a comment. Can you make sure to go over the style guide (linked in the PR description)?

Reply:
also there's a typo here.